### PR TITLE
manolo arc d11 11_1 to 11_1_2

### PR DESCRIPTION
**Pull request by nolo16**
nightmare files coming up next :sacchincopium:

### CODE REVIEW

File: script/Arcueid/Day 11/11_00_ARC11_1.txt [hunk 1]
```diff
@@ -121,17 +124,18 @@ It's not that I lost myself when I saw her golden eyes.
 [sha:1d71a1f593f73fccfc992c40c599cd49d627afab]{
 -- Page 8, Offset 29498.
 -- いつからか本気になっていただけ。
-It's that at some point I started getting serious.
+// i have no idea
+It's just that at some point, I started taking her seriously.
```
Comment (yk1512) [inline script/Arcueid/Day 11/11_00_ARC11_1.txt:128]: I started being serious about her

File: script/Arcueid/Day 11/11_00_ARC11_1.txt [hunk 2]
```diff
@@ -121,17 +124,18 @@ It's not that I lost myself when I saw her golden eyes.
...
 -- Page 9, Offset 29499.
 -- 理由を言葉にしないまま、本気であいつに惹かれていた。
-Without really knowing why, I'd becoming completely charmed by her.
+Without really knowing why, I was completely charmed by her.
 }
 [sha:2479d5de83eecc9b9fee0c2bfd9bf9a90e7c4431]{
 -- Page 9, Offset 29500.
 -- もうずっと以前から―――おそらくは、あいつが俺のような人でなしを許すと言ったあの時から―――ロアの事なんて関係なく、あいつ自身にいかれていたんだ。
-I've been that way for a long time now—probably from the moment she said she'd forgive someone like me—regardless of Roa. I've been broken by her.
+I've been this way for a long time now. Probably from the moment she said she'd forgive someone like me, regardless of Roa, I've been captivated by her.
```
Comment (yk1512) [inline script/Arcueid/Day 11/11_00_ARC11_1.txt:138]: semicolon after 'now', period after 'me'.

File: script/Arcueid/Day 11/11_00_ARC11_1.txt [hunk 3]
```diff
@@ -211,17 +215,17 @@ Everyone in attendance is talking about how tomorrow is a day off.
...
 -- Page 15, Offset 29516.
 -- 「そっか。今日は金曜日だったっけ」
-"Huh. I guess today is friday."
+"Huh. I guess today is Friday."
 }
 [sha:2109caf7d5ced7332aeb9bf776003c17801d4add]{
 -- Page 15, Offset 29517.
 -- ここ数日間、あいつに振りまわされていた事もあり、曜日の感覚が薄れていた。
-Over the past few days Arcueid has been pushing me so hard that my sense of what day it has been fading.
+Over the past few days Arcueid has been pushing me so hard that my sense of time has been fading.
```
Comment (yk1512) [inline script/Arcueid/Day 11/11_00_ARC11_1.txt:223]: comma after days

File: script/Arcueid/Day 11/11_00_ARC11_1.txt [hunk 4]
```diff
@@ -401,17 +405,18 @@ The laid-back math teacher said absent-mindedly.
 [sha:149ca5da704b4f1669dc604b8cd1ccd3546f71dd]{
 -- Page 27, Offset 29554.
 -- 「一週間もたないとか自由にも程がある」
-"She couldn't even last a week"
+// not too sure here
+"She couldn't even last a week. Jeez, talk about too much freedom."
```
Comment (yk1512) [inline script/Arcueid/Day 11/11_00_ARC11_1.txt:409]: talk about being too free-spirited

**Review by yk1512 — APPROVED** [review]

**Review by Avuxo — APPROVED** [review]

**Pull request merged**